### DISCUSSION TITLE
Create folders / custom exercises from JSON

### DISCUSSION THREAD

**Discussion by haydeninghem** [SteveG/underthebar#5] [Ideas]
Hey this is a cool project, I figured id ask first before trying to dive in: 

Is it possible to create folders and new exercises thru the routine creator? if this functionality does not already exist, do you have any ideas from the available APIs if it would be possible to implement?

**Comment by SteveG** [comment]
I haven't bothered at the moment, the routines list I have is somewhat ordered by folder though. Of course it would be possible. I had a look at what happens:

> To create make up some json.
>
>{
>  "folder": {
>    "title": "fish"
>  }
>}
>
>POST the above json to https://api.hevyapp.com/routine_folder
>
>GET https://api.hevyapp.com/routine_folders
>
>To update need to use the folderId that was returned:
>
>{
>  "folderId": 190526,
>  "title": "big_fish"
>}
>
>PUT the above json to https://api.hevyapp.com/routine_folder
>
>GET https://api.hevyapp.com/routine_folders
>
>To Delete use the folderId:
>
>DELETE https://api.hevyapp.com/routine_folder/190526
